src/GraphQL/Utilities/SchemaPrinter.cs

```diff
@@ -386,6 +400,48 @@ public string PrintDescription(string description, string indentation = "", bool
             return desc;
         }
 
+        public string PrintDescription(string description, string indentation = "", bool firstInBlock = true)
```

Comment:
Reading this, I do not cease to remember the parser code, where it saved every allocation.